README.md

```diff
@@ -51,27 +51,27 @@ class app_widgetContainer {
 }
 ```
 
-For methods, follow the standard Apple convention of referring to the first parameter in the method name:
+For methods, when the first argument is part of a prepositional phrase, make the preposition the first label and use the actual argument name inside the method definition:
 
 ```swift
 class Guideline {
-  func combineWithString(incoming: String, options: Dictionary?) { ... }
-  func upvoteBy(amount: Int) { ... }
+  func combine(with incoming: String, options: Dictionary?) { ... }
+  func upvote(by amount: Int) { ... }
```

Comment:
I tend to side with @pwc3 in that some redundancy can sometimes make a big difference when reading code. A similar point is addressed under the Swift API Design Guidelines: https://swift.org/documentation/api-design-guidelines/#omit-needless-words section: "Compensate for weak type information".
While not the same case, it's similar in spirit.